### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Agattoss**

### CODE REVIEW

File: apps/toy-stories/src/app/app.module.ts [hunk 1]
```diff
@@ -4,9 +4,16 @@ import { BrowserModule } from '@angular/platform-browser';
...
+import { HeaderComponent } from './homepage/cointeiners/header/header.component';
+
 
 @NgModule({
-  declarations: [AppComponent, NxWelcomeComponent],
+  declarations: [
+    AppComponent,
+    NxWelcomeComponent,
+    HeaderComponent,
+
```
Comment (JezynkaMJ) [inline apps/toy-stories/src/app/app.module.ts:15]: remove white space, best use prettier formatting on file

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      </ul>
+    </nav>
+  </div>
+  <div class="login">
+      <button type="button" class="login__btn">Login</button>
+      <button type="button" class="login__btn">Sign up</button>
+
+  </div>
+
+</div>
```
Comment (JezynkaMJ) [inline apps/toy-stories/src/app/homepage/cointeiners/header/header.component.html:19]: format file with prettier

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+  flex: 4;
+
+  .nav {
+    width: auto;
+    display: flex;
+    width: 100%;
+    justify-content: space-evenly;
+
+    &__a {
+      text-decoration: none;
```
Comment (JezynkaMJ) [inline apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss:39]: two times text decoration

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+      color: #41403e;
+      cursor: pointer;
+      display: inline-block;
+      font-family: Neucha, sans-serif;
+      font-size: 1rem;
+      font-weight: bold;
+      line-height: 23px;
+      outline: none;
+      padding: 0.75rem;
+      text-decoration: none;
```
Comment (JezynkaMJ) [inline apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss]: here again

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+      list-style-type: none;
+      display: flex;
+      justify-content: space-evenly;
+      align-items: center;
+      width: 80%;
+      white-space: nowrap;
+
+
+
+    }
```
Comment (JezynkaMJ) [inline apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss:75]: unnecessary white space. Format with prettier

File: package.json [hunk 6]
```diff
@@ -23,6 +23,7 @@
     "@nestjs/core": "^9.0.0",
     "@nestjs/platform-express": "^9.0.0",
     "@nrwl/angular": "15.0.10",
+    "bootstrap": "^5.2.2",
```
Comment (JezynkaMJ) [inline package.json]: are we using bootstrap?

**Review by JezynkaMJ — APPROVED** [review]

**Comment by JezynkaMJ** [comment]
please squash commits before merging them. You can do that here on git hub.
Merge pull request => Sqash and merge

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.html (hunk 2 above)
Comment (Agattoss) [reply apps/toy-stories/src/app/homepage/cointeiners/header/header.component.html:19]: Done

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss (hunk 3 above)
Comment (Agattoss) [reply apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss:39]: Done

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss (hunk 4 above)
Comment (Agattoss) [reply apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss]: Done

File: apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss (hunk 5 above)
Comment (Agattoss) [reply apps/toy-stories/src/app/homepage/cointeiners/header/header.component.scss:75]: Done

**Pull request merged**